packages/plans-grid/src/plans-interval-toggle/style.scss

```diff
@@ -0,0 +1,106 @@
+@import '~@automattic/typography/styles/variables';
+@import '../variables';
+
+.plans-interval-toggle {
+	display: flex;
+	justify-content: center;
+
+	.segmented-control {
+		-webkit-tap-highlight-color: transparent;
+
+		background-color: #f2f2f2;
+		border-radius: 6px; /* stylelint-disable-line scales/radii */
+		height: auto;
+		overflow: hidden;
+	}
+
+	&--monthly .segmented-control {
+		// Make space for popover on mobile
+		margin-bottom: 60px;
+
+		@include break-medium {
+			margin-bottom: 0;
+		}
+	}
+
+	.segmented-control__item .segmented-control__link {
+		border-radius: 5px; /* stylelint-disable-line scales/radii */
+		margin: 2px;
+		padding: 0;
+	}
+
+	.segmented-control__item:not( .is-selected ) .segmented-control__link {
+		background-color: transparent;
+		border: 0.5px solid transparent;
+	}
+
+	.segmented-control__item.is-selected .segmented-control__link {
+		background-color: var( --studio-white );
+		border: 1px solid #f5f5f5;
+		box-shadow: 0 3px 8px rgba( 0, 0, 0, 0.12 ), 0 3px 1px rgba( 0, 0, 0, 0.04 );
+	}
+
+	.segmented-control__item:not( .is-selected ) .segmented-control__link,
+	.segmented-control__item.is-selected .segmented-control__link {
+		&:focus,
+		&:hover {
+			border-color: var( --studio-gray-10 );
+		}
+	}
+
+	.segmented-control__text {
+		display: inline-flex;
+		align-items: center;
+		justify-content: center;
+
+		height: 32px;
+		min-width: 102px;
+		padding: 0 16px;
```

Comment:
Why do we need padding here? And should it affect the tooltip's alignment? This is what happens when I toggle it:  

https://user-images.githubusercontent.com/12104589/104573546-4da80d00-5655-11eb-8cca-ced66acea501.mov

Reply:
It's there for 2 reasons:

- the toggle `items` have a minimum width of `102px`, but if the text gets longer (for example, when translated to another language), that padding is there to ensure some white space around it
- and yes, as you pointed out, is also affects the position of the popover. It's hard to replicate exactly the same positioning and spacing of the `/start` flow, because I had to use a different Popover component (as explained in the PR description). This component positions itself within the parent element, and so adding the padding there also helped nudging the popover to the right

<img width="606" alt="Screenshot 2021-01-14 at 10 48 55" src="https://user-images.githubusercontent.com/1083581/104574257-1b4adf80-5656-11eb-8a53-fd8fa4b7f4d5.png">

Reply:
Thanks for elaborating! It might make sense to move the padding to its parent `<a/>` since this is already a block element which might makes life (regarding padding) a bit easier?

Reply:
Sorry for the spam but why do we need a `min-width` and fixed `height`? And how does that affect the (mobile) viewports?

Reply:
The `min-width` and `height` are following the design specs.

In particular for the min-width, both toggle "items" in the design have the same width, even if they don't have the same text. So I decided to add a `min-width` rule, which will make the items the same width, unless the copy gets longer

Reply:
I tried moving padding to the `<a>` parent, but the positioning of the popover doesn't change. This is because the Popover gets anchored to the `<span>` element, and so, moving the padding from the `span` to the parent `a` doesn't affect the actual width of the `span` contents.

The only way to push the popover position more to the right is to remove the `padding` completely, but that will have negative effects on how longer labels will look like in the toggle (in this example, notice the lack of space to the left of "Some" and to the right of "here")

<img width="568" alt="Screenshot 2021-01-14 at 12 17 08" src="https://user-images.githubusercontent.com/1083581/104584195-7be01980-5662-11eb-8815-efddb92e6b83.png">

Reply:
Update: I actually managed to have this working by wrapping the content of the Segmented Control Items into an additional span. This is probably achieving the best of both worlds

<img width="891" alt="Screenshot 2021-01-14 at 15 36 35" src="https://user-images.githubusercontent.com/1083581/104605150-6b3d9c80-567e-11eb-82ba-8e4f722b9d98.png">